modules/install/pages/install.adoc

```diff
@@ -1,71 +1,538 @@
+
+* Configure the RHEL9 server to use Java 17 if you have multiple Java versions installed.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo alternatives --config java
+There is 1 program that provides 'java'.
+
+  Selection    Command
+-----------------------------------------------
+*+ 1           java-17-openjdk.x86_64 (/usr/lib/jvm/java-17-openjdk-17.0.10.0.7-2.el9.x86_64/bin/java)
+
+Enter to keep the current selection[+], or type selection number: 1
+----
+
+Press the number corresponding to the Java 17 version, and the press `Enter`. Make a note the Java 17 location mentioned in the above output, in this case `/usr/lib/jvm/java-17-openjdk-17.0.10.0.7-2.el9.x86_64`. We shall need this value in next step.
+
+* Edit the `/home/jboss/EAP-8.0.0/bin/init.d/jboss-eap.conf` file and remove the `#` in front of each of the following variable and update the following environment variables. Substitude the `JAVA_HOME` value to `/usr/lib/jvm/java-17-openjdk-17.0.10.0.7-2.el9.x86_64/bin/java` or the value you got from the previous step. Similarly, substitute other environment variables as per your environment.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ cat /home/jboss/EAP-8.0.0/bin/init.d/jboss-eap.conf
+JAVA_HOME="/usr/lib/jvm/java-17-openjdk-17.0.10.0.7-2.el9.x86_64"
+JBOSS_HOME="/home/jboss/EAP-8.0.0"
+JBOSS_USER=jboss
+JBOSS_MODE=standalone
+JBOSS_CONFIG=standalone.xml
+JBOSS_CONSOLE_LOG="/var/log/jboss-eap/console.log"
+----
+
+* Copy the file jboss-eap.conf file to the /etc/default directory by running the following command:
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo cp /home/jboss/EAP-8.0.0/bin/init.d/jboss-eap.conf /etc/default/jboss-eap.conf
+----
+
+* Copy the jboss-eap-rhel.sh file to the /etc/init.d directory with jboss-eap as the file name, and change its permission to be executable by running the following commands.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo cp /home/jboss/EAP-8.0.0/bin/init.d/jboss-eap-rhel.sh /etc/init.d/jboss-eap
+[vagrant@server ~]$ sudo chmod 755 /etc/init.d/jboss-eap
+----
+
+* Reload the systemd daemon using the following commands.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo systemctl daemon-reload
+----
+
+* Enable the `jboss-eap` service to start on system boot.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo systemctl enable jboss-eap
+----
+
+* To verify if the setup was successful, run:
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo systemctl start jboss-eap
+----
+or reboot the system.
+
+* Navigate to the JBoss EAP console at `http://localhost:8080` using firefox browser and confirm JBoss EAP is running as shown in the below screenshot.
++
+image::ui.png[align="center"]
+
+== Lab 5: Uninstalling JBoss EAP service on RHEL
+
+*Prerequisites*:
+You have JBoss EAP installed, or your have followed Lab 4
+
+*Outcome*: You remove an instance of JBoss EAP and any services associated with it.
```

Comment:
These steps only remove the service. Rename this section as "Uninstalling JBoss EAP" and then first show them how to uninstall JBoss EAP using the uninstallation GUI wizard followed by how to remove the service. FOr the other options write a sentence or two about simply deleting the folder under /opt - does the installation manager shell scriptn offer an option to uninstall just the like GUI JAR installer?

Reply:
Added the section